src/Microsoft.AspNet.Mvc.DataAnnotations/AttributeAdapterBase.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNet.Mvc.ModelBinding.Validation;
+using Microsoft.Extensions.Localization;
+
+namespace Microsoft.AspNet.Mvc.DataAnnotations
```

Comment:
Place this class in the `Microsoft.AspNet.Mvc.ModelBinding.Validation` namespace and remove above `using`.